packages/internal/src/build/babel/api.ts

```diff
@@ -0,0 +1,87 @@
+import fs from 'fs'
+import path from 'path'
+
+import type { TransformOptions, PluginItem } from '@babel/core'
+import babelRequireHook from '@babel/register'
+
+import { getPaths } from '../../paths'
+
+export const getApiSideBabelPlugins = () => {
+  const rwjsPaths = getPaths()
+  // Plugin shape: [ ["Target", "Options", "name"] ],
+  // a custom "name" is supplied so that user's do not accidently overwrite
+  // Redwood's own plugins.
+  const plugins: TransformOptions['plugins'] = [
+    ['@babel/plugin-transform-typescript', undefined, 'rwjs-babel-typescript'],
+    [
+      require('@redwoodjs/core/dist/babelPlugins/babel-plugin-redwood-src-alias')
+        .default,
```

Comment:
@peterp 

Notice I had to add `.default` because I think.... with the datamigrate hook was doing the transform for us (accidentally)

Reply:
Did you manage to fix this by making it inline? Super weird behaviour, it should be ignore node_modules.

Reply:
Not sure what you mean by inline - as in require them before the plugin block?

Reply:
You mentioned that you changed something in datamigrate when you registered the hook, is this still a problem?

Reply:
I think it was the opposite actually - because the cli was registering the hooks, we didn't need to do the `.default`. It is very strange to me as well...